### PR TITLE
Un 506 content warning

### PR DESCRIPTION
**Pull request by William-Blackie**
Ticket URL: Ticket [here](https://national-archives.atlassian.net/jira/software/c/projects/UN/boards/75?modal=detail&selectedIssue=UN-506)

## About these changes

This updates the styling and positioning for the content warning element for the following pages: 
- ArticlePage (aka Story)
- RecordArticlePage
- HighlightGalleryPage

Please note: The ticket does specify a blog page also but this does not yet exist.

I've tried were possible to match the new designs in terms of layout but there are some small differences as the changes required to match are far greater than the scope of this ticket.

### ArticlePage
0 0 0 0_8000_admin_pages_187_edit_preview__mode= (1)
0 0 0 0_8000_admin_pages_187_edit_preview__mode=

### RecordArticlePage
0 0 0 0_8000_admin_pages_214_edit_preview__mode= (1)
0 0 0 0_8000_admin_pages_214_edit_preview__mode=

### HighlightGalleryPage
0 0 0 0_8000_admin_pages_221_edit_preview__mode= (1)
0 0 0 0_8000_admin_pages_221_edit_preview__mode=

## How to check these changes

Open the various page types, enable showing the warning in the admin. Inspect the FE and make sure the default text or the custom text shows. 

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer.
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes.
- [x] Ensured that PR includes only commits relevant to the ticket.
- [x] Waited for all CI jobs to pass before requesting a review.
- [x] Added/updated tests and documentation where relevant.

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: sass/includes/_content-warning.scss [hunk 1]
```diff
@@ -1,26 +1,56 @@
...
+    margin-bottom: 3rem;
+  }
+
+  &--normalise-margin {
+    // normalise full-wdith mobile display
+    margin-left: -15px;
+    margin-right: -15px;
+  }
+
+  &--normalise-article-margin {
```
Comment (William-Blackie) [inline sass/includes/_content-warning.scss:23]: Note: Both of these are work-arounds for the current widths of heros/ body widths. You can see I'm trying to normalise the content warning to be full-width to the page as best as we can.

File: sass/includes/_generic-intro.scss [hunk 2]
```diff
@@ -30,7 +30,6 @@
   }
 
   &__paragraph {
-    max-width: 50rem; // 800px for decent reading line length
```
Comment (William-Blackie) [inline sass/includes/_generic-intro.scss:33]: Note: No longer needed in the latest designs.

File: templates/collections/highlight_gallery_page.html [hunk 3]
```diff
@@ -16,40 +16,6 @@
...
-                        {{ page.featured_article.title }}
-                    </h2>
-                    <p>{{ page.featured_article.teaser_text }}</p>
-                    <a class="tna-button--dark" href="{% pageurl page.featured_article %}">Read <span class="sr-only">about {{ page.featured_article.title }}</span></a>
-                </div>
-            </div>
-        </div>
-    {% endif %}
-
-
```
Comment (William-Blackie) [inline templates/collections/highlight_gallery_page.html:54]: Note: I've added the new template as the old blows out page widths with images of a certain size.

**Comment by JohnHeeryTNA** [comment]
Hi @William-Blackie - we have a few merge conflicts on this one, could you have a look please.

**Review by JohnHeeryTNA — APPROVED** [review]
approving changes

**Pull request merged**